### ISSUE TITLE
glTF Addon can not be enabled with blender 2.8 Release Cadidate

### ISSUE THREAD

**Issue by naotaro0123** [KhronosGroup/glTF-Blender-IO#592]
**Describe the bug**
The following error appears when enabling the glTF add-on.
```
Traceback(most recent call last):
File "/Applications/blender-2.80.0/Blender.app/Contents/Resources/2.80/scripts/modules/addon_util.py", line 351, in enable mod = __import__(module_name)
File "/Applications/blender-2.80.0/Blender.app/Contents/Resources/2.80/scripts/addons/io_scene_gltf2/__init_py", line 63, in <module> from io_scene_gltf2.io.exp import gltf2_io_draco_comression_extension
File "/Applications/blender-2.80.0/Blender.app/Contents/Resources/2.80/scripts/addons/io_scene_gltf2/io/exp/gltf2_io_draco_compression_extension.py", line 17, in from ctypes import *
File "/Applications/blender-2.80.0/Blender.app/Contents/Resources/2.80/python3.7/ctypes/__init__.py", line 543, in <module> _reset_cache()
File "/Applications/blender-2.80.0/Blender.app/Contents/Resources/2.80/python3.7/ctypes/__init__.py", line 273, in _reset_cache CFUNCTYPE(c_int)(lambda: None)
MemoryError
```

**To Reproduce**
1. Open the blender app. Menu does not have gltf menu in [File]-[Export / Import]

**Expected behavior**
glTF menu is displayed.

**Screenshots**
I get the following error when enabling glTF 2.0 Addon.
<img width="902" alt="screenshot01" src="https://user-images.githubusercontent.com/3956770/61214965-52c88b00-a744-11e9-86fd-2ea1a0eaa6d3.png">

**Version** 
- OS: MacOS Mojave (10.14.5)
- CPU: 2.5 GHz Intel Core i5
- Memory: 8GB 1600 MHz DDR3
- GPU: Intel HD Graphics 4000 1536MB
 - Blender blender2.8 Release Cadidate

**Additional context**
Blender 2.8 Beta used glTF Addon.

**Comment by naotaro0123** [comment]
Sorry, I tried on the following Mac. glTF Addon worked without problems.
Is the cause insufficient memory?

**Version**
- OS:MacOS High Sierra(10.13.6)
- CPU: 3.1GHz Intel Core i7
- Memory: 16GB 1867 MHz DDR3
- GPU: Intel Iris Graphics 6100 1536MB

**Comment by emackey** [comment]
@UX3D-nopper Can you look at this?  It's Draco-related and I don't have access to MacOS to test it.

**Comment by ZanQdo** [comment]
Uh oh! Getting the same on Windows 10

https://pastebin.com/CcTNQqj2

**Comment by LazyDodo** [comment]
The initial release of the RC for mac had code sign issues and has been silently updated (which i admit is not great) try re-downloading it from https://download.blender.org/release/Blender2.80/ and see if that improves the situation.

**Comment by naotaro0123** [comment]
Thank you for updated!
After try re-downloading it, the problem is solved with MacOS Mojave(10.14.5).
